.github/workflows/ballerina-central-stats.yml

```diff
@@ -0,0 +1,30 @@
+name: Dashboard Extractor
+run-name: Testing out GitHub Actions
+on: [push]
```

Comment:
Is this trigger correct?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-release/pull/2576/commits/b351f15e1d0a2d11b1ce8f7b32e100830175825d